lib/graphql-anycable.rb

```diff
@@ -20,6 +21,10 @@ def self.use(schema, **options)
       schema.use GraphQL::Subscriptions::AnyCableSubscriptions, **options
     end
 
+    def self.stats(include_subscriptions: false)
+      AnyCable::Stats.new(redis: redis, config: config, include_subscriptions: include_subscriptions).collect
```

Comment:
What's the purpose of passing `redis` and `config` here? They're both global singletons, we can call them directly form the Stats class.

That would allow us to simplify the method as follows:

```ruby
def self.stats(**options)
  Stats.new(**options).collect
end
```

Reply:
yea, you are right, thanks
fixed!